src/viam/sdk/tests/test_robot.cpp

```diff
@@ -129,28 +157,23 @@ BOOST_AUTO_TEST_CASE(test_get_status) {
             // get all resource statuses
             auto statuses = client->get_status();
 
-            auto status_strs = vec_to_string_util(statuses);
-            auto mock_strs = vec_to_string_util(mock_statuses);
-
             // ensure we get statuses for all resources, and that they are as expected.
             BOOST_CHECK_EQUAL(statuses.size(), 3);
-            BOOST_TEST(status_strs == mock_strs, boost::test_tools::per_element());
+            BOOST_TEST(statuses == mock_statuses, boost::test_tools::per_element());
 
             // get only a subset of status responses
             auto names = mock_resource_names_response();
-            std::vector<ResourceName> some_names{names[0], names[1]};
+            std::vector<Name> some_names{names[0], names[1]};
             auto some_statuses = client->get_status(some_names);
-            auto some_status_strs = vec_to_string_util(some_statuses);
-
             // ensure that we only get two of the three existing statuses
-            BOOST_CHECK_EQUAL(some_status_strs.size(), 2);
+            BOOST_CHECK_EQUAL(some_statuses.size(), 2);
 
             // unfortunately the sorting is a bit odd so we end up with a mismatch of index,
             // but this ensures that the statuses we received do exist in the mocks, as
             // expected.
-            std::vector<std::string> some_mock_strs{mock_strs[1], mock_strs[2]};
+            std::vector<RobotClient::status> some_mock_statuses{mock_statuses[0], mock_statuses[2]};
```

Comment:
The fact that we've seen a change in the order of the statuses is a bit weird! Looking at `mock_status_response`, it looks like you caught and corrected an issue where we were setting statuses out of order (first grabbing element 1, _then_ element zero), thanks for catching that! But, we also changed the order we put the elements in the list. Can we try to create some consistency so that `some_statuses` is equal to `some_mock_statuses{mock_statuses[0], mock_statuses[1]}`? Then we can delete that weird comment above and set that whole issue to bed :)